dandi/tests/test_upload.py

```diff
@@ -30,14 +32,16 @@ def test_upload(local_docker_compose_env, monkeypatch, organized_nwb_dir2):
     client = girder.get_client(local_docker_compose_env["instance"].girder)
     for dname in [dirname1, dirname2]:
         with pytest.raises(girder.GirderNotFound):
-            girder.lookup(client, collection_drafts, path=f"{dandi_id}/{dname}")
+            girder.lookup(client, collection_drafts, path=op.join(dandi_id, dname))
```

Comment:
`girder.lookup()` passes `path` to `resourceLookup()`, [the documentation for which](https://girder.readthedocs.io/en/latest/python-client.html#girder_client.GirderClient.resourceLookup) states that "The path must be an absolute Unix path".  All of the changes to `girder.lookup()` in this PR need to be undone.

Reply:
d'oh -- right, indeed all paths to girder should be the POSIX ones.  dropping that 24820e0fc316128fbf85c4dc384c09b31d6733ee commit